### PR TITLE
fixed equal @ calc display bug

### PR DESCRIPTION
**Pull request by nayelyrodarte**

### CODE REVIEW

**Comment by kermitoid** [comment]
<h3>🐸&nbsp;&nbsp;Thanks for your pull request, ribbit!</h3>Go fix yourself a piña colada while the checks pass and someone reviews your code. 🍍<br></br><h3>📑&nbsp;&nbsp;You forgot to update the <code>CHANGELOG</code>!</h3>You've modified <b>Phantom</b>. Please record your changes to <a href="https://github.com/sidiousvic/phantom/blob/production/README.md"><code>CHANGELOG.md</code></a>.<br></br><!--
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f6145ba4c35ad7721cd59ce9a781d9dbe1b125f9
</p>

File: examples/calculator/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import phantom from "../../phantom.ts";
+import phantom from "../../src/phantom";
```
Comment (sidiousvic) [inline examples/calculator/index.js:1]: Thanks for reminding me of this 👍🏼

File: examples/calculator/redux.js [hunk 2]
```diff
@@ -24,13 +24,23 @@ function reducer(state = initialState, action) {
...
+        };
+      } else {
+        return {
+          ...state,
+          screen: [state.screen.join(""), action.operator],
+          n1,
+          operator: action.operator,
+          state: "calculating",
+        };
+      }
```
Comment (sidiousvic) [inline examples/calculator/redux.js:43]: Sweet!  🐍👍🏼

**Comment by sidiousvic** [comment]
Gracias, Naye!

**Pull request merged**